packages/alert/src/close-icon.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import {Button} from '@smashing/button'
+
+const CloseButton = styled(Button)`
+  margin-left: auto;
+  box-sizing: content-box;
+  height: auto;
```

Comment:
Control height with a height prop on button

Reply:
Unfortunately `height: auto` is needed for proper rendering as flex element